tox.ini

```diff
@@ -1,60 +1,30 @@
 [tox]
 envlist =
-    py27,py34,py35,py36,py37,pypy,
-    py27-pyramid{12,13,14,15,16,17,18,19},
+    py34,py35,py36,py37,py38,py39,pypy3,
     docs,
-    {py2,py3}-cover,coverage
+    coverage
 
 [testenv]
 # Most of these are defaults but if you specify any you can't fall back
 # to defaults for others.
 basepython =
-    py27: python2.7
     py34: python3.4
     py35: python3.5
     py36: python3.6
     py37: python3.7
     py38: python3.8
-    pypy: pypy
+    py39: python3.9
     pypy3: pypy3
-    py2: python2.7
-    py3: python3.5
-
-deps =
```

Comment:
These are still important as we want to test this against multiple versions of Pyramid and be able to do so easily.

Reply:
@bertjwregeer, @mmerickel said dropping 2.7 was OK: https://github.com/Pylons/pyramid_mailer/pull/96#issuecomment-830315642 

Would you please clarify what should @jon-betts do for this PR? Thank you!

Reply:
I am not talking about Py2.7 here though. I am talking about the deps specifying different versions of Pyramid. Which is the first one that introduced Python 3 support? We should support testing against that.